conf/base_config.yaml

```diff
@@ -163,4 +163,14 @@ predict:
     encoder: ${encoder}
     weight_save_path: ${paths.pretrained_models}/sdss.pt
     device: "cuda:0"
-    crop: [0, -1]
+    crop:
+        if_crop: true
+        left_upper_corner: [160, 160]
+        width: 640
+        height: 640
+    plot:
+        if_plot: true
+        width: 1000
+        height: 1000
+        out_file_name: predict.html
+    if_simulated: false
```

Comment:
Same here, it's not so self-explanatory what `if_simulated` means. Would it make senes to call it `is_simulated`?